rlw.sh

```diff
@@ -174,28 +175,29 @@ main () {
 		--title='Roblox Linux Wrapper '"$rlwversion"'-'"$branch"'' \
 		--window-icon="$RBXICON" \
 		--width=480 \
-		--height=230 \
-		--cancel-label='Quit' \
+		--height=256 \
+		--cancel-label='Exit' \
 		--list \
-		--text 'What option would you like?' \
+		--text 'What would you like to do on ROBLOX?' \
 		--radiolist \
 		--column '' \
 		--column 'Options' \
 		TRUE 'Play Roblox' \
-		FALSE 'Play Roblox (Legacy Mode)' \
-		FALSE 'Roblox Studio' \
+		FALSE 'Play Roblox (Basic Mode)' \
+		FALSE 'Develop on Roblox (Roblox Studio)' \
 		FALSE 'Reinstall Roblox' \
-		FALSE 'Uninstall Roblox' 2>/dev/null)
+		FALSE 'Uninstall Roblox' 2>/dev/null \
+		FALSE 'Visit the GitHub page')
```

Comment:
The `2>/dev/null` has to be after the last entry, in order to silence GTK-Warning errors.